app/views/layouts/_header.html.erb

```diff
@@ -9,6 +9,7 @@
           <li><%= link_to 'Features', features_path %></li>
           <li><%= link_to 'Subscriptions', subscriptions_path %></li>
           <li><%= link_to 'Transactions', transactions_path %></li>
+          <li><%= link_to 'Usages', usages_path %></li>
```

Comment:
why a global path to usages?
Shouldn't it be dependent with the plan subscription?

Reply:
This is to view all usages.

Reply:
Changed to subscriptions path